docroot/themes/custom/uids_base/templates/layout/header--homepage.html.twig

```diff
@@ -6,11 +6,12 @@
         {{ page.action_menu }}
     {% endif %}
 
-    {% if uiowa_search_form %}
-        {% include '@uids_base/uids/global/search-toggle.html.twig' with {
-            search_form: uiowa_search_form
-        } %}
-    {% endif %}
+  {% if page.search %}
+    {% include '@uids_base/uids/global/search-toggle.html.twig' with {
+      'search_form': page.search,
+      'under_search': page.under_search,
+    } %}
+  {% endif %}
```

Comment:
It's hard to tell for sure, but the indentation seems off for this section compared to the sections above and below it.